src/components/Book.js

```diff
@@ -13,14 +14,40 @@ function Book({
   };
 
   return (
-    <div className="bookDetails">
-      <span className="bookCategory">{category}</span>
-      <span className="bookTitle">{title}</span>
-      <span className="bookAuthor">{author}</span>
-      <button className="removeBook" type="button" onClick={handleRemoveClick}>
-        Remove
-      </button>
+    <div className={styles.contain}>
+      <div className={styles.bookDetails}>
+        <span className={styles.bookCategory}>{category}</span>
+        <span className={styles.bookTitle}>{title}</span>
+        <span className={styles.bookAuthor}>{author}</span>
+        <div className={styles.containButtons}>
+          <button className={styles.smallButtons} type="button">Comments</button>
+          <button className={`${styles.vertical} ${styles.smallButtons}`} type="button" onClick={handleRemoveClick}>
+            Remove
+          </button>
+          <button className={`${styles.vertical} ${styles.smallButtons}`} type="button">Edit</button>
+        </div>
+      </div>
+      <div className={styles.otherSide}>
+        <div className={styles.progress}>
+          <div className={styles.progressCircle} />
+          <div className={styles.progressOverlay} />
+          <div className={styles.progressValue} />
```

Comment:
- [ ] Please find out a good circle progress bar that matches the one on the design given because currently, this one doesn't look quite good. There are a lot of packages out there that can help you. For simplicity, you can even try this one [here](https://www.npmjs.com/package/react-circular-progressbar)
Your design
![image](https://user-images.githubusercontent.com/90222110/226118829-1d2da4ae-64dd-4159-b1d7-ce943547b618.png)
Given design
![image](https://user-images.githubusercontent.com/90222110/226118852-ad50df0e-7101-441e-bd14-4dd5f34099d8.png)